lib/edamam.rb

```diff
@@ -0,0 +1,52 @@
+class Edamam
+  class Edamam < StandardError; end
+
+  BASE_URL = "https://api.edamam.com/search?"
+
+  attr_reader :search_item, :uri
+
+  def initialize(search_item, uri)
+    @search_item = search_item
+    @uri = fix_uri
+  end
+
+  def self.all_search(search_item)
+    query_params = {
+      "q" => search_item,
+      "id" => ENV["edamam_id"],
+      "key" => ENV["edamam_key"],
```

Comment:
Not a big deal, but typically environment variables are named using all caps, so these would be `EDAMAM_ID` and `EDAMAM_KEY`. Think of it like a Ruby constant.